doc/man7/EVP_PKEY-SM2.pod

```diff
@@ -15,6 +15,23 @@ both signature and encryption schemes via the EVP interface.
 When doing the B<SM2> signature algorithm, it requires a distinguishing identifier
 to form the message prefix which is hashed before the real message is hashed.
 
+=head2 Common SM2 parameters
+
+SM2 uses the parameters defined in L<EVP_PKEY-EC(7)/Common EC parameters>.
+The following parameters are different:
+
+=over 4
+
+=item "cofactor" (B<OSSL_PKEY_PARAM_EC_COFACTOR>) <unsigned integer>
+
+This parameter is ignored for B<SM2>.
+
+=item (B<OSSL_PKEY_PARAM_DEFAULT_DIGEST>) <UTF8 string>
+
+Getter that returns the default digest string "SM3".
```

Comment:
Same here as above.

Reply:
I dont see an issue - if we change what the default is then that should get documented also as a migration guide change. Ignoring the fact there is a default seems like a weird thing to do.
Or do you think this should not be documented at all maybe?

Reply:
No, my concern is that this somehow makes the value to be unchangeable in future if it is documented like this. Perhaps just adding "currently". `Gettable parameter that returns the default digest name. ("SHA256/SM3" as of OpenSSL 3.0.)`

Reply:
Adding a version helps a little I think.  We have changed defaults before and will do in the future.  I don't think it's a big deal either way.